modules/openapi-generator/src/main/resources/JavaSpring/methodBody.mustache

```diff
@@ -43,5 +43,8 @@ exchange.getResponse().setStatusCode(HttpStatus.NOT_IMPLEMENTED);
         }
         {{/-last}}
     {{/examples}}
+{{^examples}}
+        exchange.getResponse().setStatusCode({{#returnSuccessCode}}HttpStatus.OK{{/returnSuccessCode}}{{^returnSuccessCode}}HttpStatus.NOT_IMPLEMENTED{{/returnSuccessCode}});
```

Comment:
Shouldn't it be `HttpStatus.valueOf({{{statusCode}}})` instead of `HttpStatus.OK` ?

Reply:
Yes, it should be `HttpStatus.OK`  as [the status code is contained by example](https://github.com/OpenAPITools/openapi-generator/pull/1197/files#diff-48c5051e9bbbf89ae664cd5918204e52R65). The code above is in `{{^examples}}` section.